graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -507,6 +336,112 @@ def _combine_filter_selectivities(selectivities):
     return Selectivity(kind=combined_selectivity_kind, value=combined_selectivity_value)
 
 
+def get_selectivity_of_filters_at_vertex(schema_info, filter_infos, parameters, location_name):
+    """Get the combined selectivity of all filters at the vertex.
+
+    Args:
+        schema_info: QueryPlanningSchemaInfo
+        filter_infos: list of FilterInfos, filters on the location being filtered
+        parameters: dict, parameters with which query will be executed
+        location_name: string, type of the location being filtered
+        counts: float, result count that we're adjusting for filters
+
+    Returns:
+        Selectivity object
+    """
```

Comment:
I know this function can be split up into smaller ones, but it's hard to tell where before integrating inequality computations for non-uuid field filters. That's coming up next.